website/docs/how-tos/how-to-culculate-script-hash.mdx

```diff
@@ -0,0 +1,30 @@
+---
+id: how-to-culculate-script-hash
+title: How to Calculate Script Hash
+---
+
+import Tooltip from "@components/Tooltip";
+
+# How to Calculate Script Hash
+
+Script hash is calculated as follows:
+
+```
+ckbhash(molecule_encode(script))
+```
+
+Where:
+
+- <Tooltip>ckbhash</Tooltip> is the BLAKE2b hash algorithm specifically used in CKB
```

Comment:
Maybe take out the Tooltip as this sentence already explained what ckbhash is doing?